scripts/mutect2_wdl/README.md

```diff
@@ -195,7 +192,7 @@ gs://broad-dsde-methods/takuto/na12878-crsp-ice/SM-612V3.bam    gs://broad-dsde-
   "Mutect2_Multi.variants_for_contamination_index": "/data/m2/gnomad-common-biallelic-snps.vcf.idx",
   "Mutect2_Multi.is_run_orientation_bias_filter": true,
   "Mutect2_Multi.is_run_oncotator": true,
-  "Mutect2_Multi.m2_docker": "broadinstitute/gatk:1.0.0.0-alpha1.2.4",
+  "Mutect2_Multi.gatk_docker": "broadinstitute/gatk:1.0.0.0-alpha1.2.4",
```

Comment:
Can we update the example value here?  It's way out of date.  How about `broadinstitute/gatk:4.beta.5`?

Reply:
done